### PR TITLE
Replace custom icons for material icons in PageTransactions

### LINKED ISSUE #99: Remove custom icons
In the transaction list, let's use the following material icons in favour of the custom ones. This will make the UI more future proof, as we will be adding other TXs over time:

Send: arrow_up
Receive: icon=arrow_down
Swap: compare_arrows
Stake: thumb_up
Restake: thumbs_up_down
Unstake: thumb_down
Unknown: help_outline

Above are just to get started, will experiment with different icons once done.

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #99 

image

### CODE REVIEW

**Comment by mdyring** [comment]
Seems to have affected the signin icons that now have red background:
image

File: components/transactions/MessageItem.vue [hunk 1]
```diff
@@ -116,26 +112,26 @@ export default {
...
     },
     icon() {
-      try {
-        const icon = String(this.messageCaption).replace(' ', '')
-        switch (icon) {
-          case `Send`:
-            return `arrow_upward`
-          case `Receive`:
+      switch (this.message.type) {
+        case lunieMessageTypes.SEND:
```
Comment (Bitcoinera) [inline components/transactions/MessageItem.vue:116]: ah, yes, using the enum is much better :wink:

**Comment by Bitcoinera** [comment]
> Seems to have affected the signin icons that now have red background:

oops. My bad

**Comment by Bitcoinera** [comment]
Yes! These icons look definitely better!

image

**Pull request merged**